Plugins/MatrixPlugin/MatrixLogic.cs

```diff
@@ -0,0 +1,334 @@
+using Chummer;
+using Chummer.Backend.Attributes;
+using Chummer.Backend.Equipment;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Linq;
+
+namespace MatrixPlugin
+{
+    public class MatrixLogic : INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        private readonly Character _character;
+
+        public List<IHasMatrixAttributes> Persons;
+        public List<Gear> Software;
```

Comment:
Does this really need to be a `List<Gear>`? Based on how you end up using this in the rest of your code, it really, really looks much more suited to be a `HashSet<string>` of names instead.